test/e2e/cp_test.go

```diff
@@ -141,6 +141,8 @@ var _ = Describe("Podman cp", func() {
 	})
 
 	It("podman cp stdin/stdout", func() {
+		SkipIfRemote()
+		Skip("Looks like SkipIfRemote() is not working")
```

Comment:
[For real](https://storage.googleapis.com/cirrus-ci-5385732420009984-fcae48/artifacts/containers/libpod/6438322283216896/html/integration_test.log.html#t--podman-cp-stdin-stdout--1). 

The code in [libpod_suite_remote_test.go](https://github.com/containers/libpod/blob/master/test/e2e/libpod_suite_remote_test.go#L22) looks OK to me. I don't understand this.

Reply:
Um. It has to just be this one case, because we'd have a lot more errors if it wasn't working everywhere...

Reply:
IKR. But look at the function definition for `SkipIfRemote()`:
```go
    ginkgo.Skip("This function is not enabled for remote podman")
```
Then look at that giant log file again. Do an in-page search for the string 'enabled for'.

I'm really confused.

Reply:
Is libpod_suite_remote_test.go  being used?

Reply:
To my eye -- which of course is not well-versed in Go -- it has the right build tags:
```go
// +build remoteclient
```
I still don't understand why `not enabled for remote podman` doesn't appear in the test log. Many of the tests calling `SkipIfRemote()` are tagged `!remoteclient` (presumably for the v2 transition) -- but not all of them.

The one thing I don't actually see is `remoteclient` anywhere in Makefile. There's `remote`, and I recall discussion in scrum about a difference between those, but wasn't/amn't educated enough to understand the difference.